### PR TITLE
CandleStick Charts working

### PR DESCRIPTION
**Pull request by eymericboyer**

### CODE REVIEW

File: app/src/androidTest/java/com/sdp/swiftwallet/UiTest/CryptoGraphActivityTest.java [hunk 1]
```diff
@@ -69,15 +80,43 @@ public void unregisterIdlingResource() {
...
+            if ((type == WindowManager.LayoutParams.TYPE_TOAST)) {
+                IBinder windowToken = root.getDecorView().getWindowToken();
+                IBinder appToken = root.getDecorView().getApplicationWindowToken();
+                if (windowToken == appToken) {
+                    //means this window isn't contained by any other windows.
+                }
+            }
+            return false;
+        }
+    }*/
```
Comment (0xdieman) [inline app/src/androidTest/java/com/sdp/swiftwallet/UiTest/CryptoGraphActivityTest.java:120]: Did you leave that on purpose ?

File: app/src/main/java/com/sdp/swiftwallet/CryptoGraphActivity.java [hunk 2]
```diff
@@ -50,27 +63,98 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+
+        //Create CandleStickChart
+        CandleStickChart candleStickChart = createCandleStickChart();
+        //get data as CandleData
+        CandleData candleData = createDataSetForCandleGraph(openTimes, openValues, highValues, lowValues, closeValues, volumeValues, closeTimes);
+        //setData
+        candleStickChart.setData(candleData);
+        candleStickChart.invalidate();
+        */
```
Comment (0xdieman) [inline app/src/main/java/com/sdp/swiftwallet/CryptoGraphActivity.java:89]: Also that ?

File: app/src/main/java/com/sdp/swiftwallet/CryptoGraphActivity.java [hunk 3]
```diff
@@ -50,27 +63,98 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        candleStickChart.requestDisallowInterceptTouchEvent(true);
+
+        XAxis xAxis = candleStickChart.getXAxis();
+        xAxis.setDrawGridLines(false);
+        xAxis.setDrawLabels(true);
+        rightAxis.setTextColor(Color.BLACK);
+        yAxis.setDrawLabels(false);
+        xAxis.setGranularity(1f);
+        xAxis.setGranularityEnabled(true);
+        xAxis.setAvoidFirstLastClipping(true);
```
Comment (0xdieman) [inline app/src/main/java/com/sdp/swiftwallet/CryptoGraphActivity.java:122]: This is kind of cumbersome, isn't there a way to configure the axis in another way ?

**Review by 0xdieman — APPROVED** [review]
I see you left a lot of comments, otherwise it seems fine !

**Review by vfaltings — APPROVED** [review]
Looks good, aside from the stuff Diego pointed out

**Pull request merged**